packages/OmegaPrint-Core.package/OPPrinter.class/instance/MethodDeclaration.with.and.and..st

```diff
@@ -1,9 +1,9 @@
 actions
-MethodDeclaration: aNode with: header and: pragmas and: code
+MethodDeclaration: aNode with: aMethodHeader and: x and: executableCode
```

Comment:
warum hast du da pragmas in x geändert?

Reply:
entsprechend der neuen Conventions habe ich das wieder rueckgaengig gemacht